tests/test_spherical_harmonics.py

```diff
@@ -40,24 +49,36 @@ def test_spherical_harmonic_n10(make_coordinates, implementation):
         delimiter=',',
         dtype=complex)
 
-    basis = sh.spherical_harmonic_basis(Nmax, coords)
+    basis = sh.spherical_harmonic_basis(n_max, coords,
+                                        phase_convention='Condon-Shortley')
 
     np.testing.assert_allclose(Y, basis, atol=1e-13)
 
 
 @pytest.mark.parametrize("implementation", ['spharpy', 'pyfar'])
-def test_spherical_harmonics_real(make_coordinates, implementation):
-    n_max = 10
-    theta = np.array([np.pi/2, np.pi/2, 0, np.pi/2], dtype=float)
-    phi = np.array([0, np.pi/2, 0, np.pi/4], dtype=float)
-    rad = np.ones(4)
+@pytest.mark.parametrize("phase_convention", [None, 'Condon-Shortley'])
+@pytest.mark.parametrize("channel_convention", ['acn', 'fuma'])
+@pytest.mark.parametrize("normalization", ['n3d', 'sn3d'])
+def test_spherical_harmonics_real(make_coordinates, implementation,
+                                  normalization, channel_convention,
+                                  phase_convention):
+    n_max = 1
+    theta = np.array([np.pi/2, np.pi/2, 0], dtype=float)
+    phi = np.array([0, np.pi/2, 0], dtype=float)
+    rad = np.ones(3, dtype=float)
 
     coords = make_coordinates.create_coordinates(
         implementation, rad, theta, phi)
 
-    reference = np.genfromtxt('./tests/data/sh_basis_real.csv', delimiter=',')
-    basis = sh.spherical_harmonic_basis_real(n_max, coords)
-    np.testing.assert_allclose(basis, reference, atol=1e-13)
+    Y = np.genfromtxt(f'./tests/data/Y_real_{phase_convention}_'
+                      f'{normalization}_{channel_convention}.csv',
+                      dtype=float,
+                      delimiter=',')
+    basis = sh.spherical_harmonic_basis_real(n_max, coords,
+                                             normalization,
+                                             channel_convention,
+                                             phase_convention=phase_convention)
+    np.testing.assert_allclose(basis, Y, atol=1e-13)
```

Comment:
Could you also for the benefit of my sanity keep a test against the old files. Just so we're sure that we don't accidentally change the default normaltion and phase conventions of the files? ;)

Reply:
Sure, I added test against your old n10 files for both, real and complex. However, the new default for Condon-Shortley is None, so we explicitly need to test against phase_convention='Condon-Shortley', not the default.. Probably we should discuss at this point if we want to have "Condon-Shortley" as the default, to be consistent with old implementations?